src/components/ui/sliders/Sliders.vue

```diff
@@ -178,6 +178,15 @@
               v-model="value2"
```

Comment:
Page is overloaded.

![image](https://user-images.githubusercontent.com/5394573/59165408-67ae6f00-8b45-11e9-91e1-33e011c0deeb.png)